src/main/java/org/sciborgs1155/robot/roller/Roller.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package org.sciborgs1155.robot.roller;
+
+import static org.sciborgs1155.robot.Constants.isReal;
+import static org.sciborgs1155.robot.roller.RollerConstants.ROLLER_MAX;
+import static org.sciborgs1155.robot.roller.RollerConstants.ROLLER_REVERSE;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Roller extends SubsystemBase {
+
+  private final RollerIO hardware;
+
+  /**
+   * Creates a Roller (Constructor).
+   *
+   * @param hardware
+   */
+  private Roller(RollerIO hardware) {
+    this.hardware = hardware;
+  }
+
+  /**
+   * Creates a real intake or a no intake.
+   *
+   * @return The constructor for a real or no intake.
+   */
+  public static Roller create() {
+    return isReal() ? new Roller(new RealRoller()) : new Roller(new NoRoller());
+  }
+
+  /**
+   * A command that sets the roller.
+   *
+   * @param voltage Voltage applied to the roller.
+   * @return A command that sets the roller.
+   */
+  private Command setRoller(double voltage) {
+    return run(() -> hardware.setRoller(voltage));
+  }
+
+  /**
+   * Runs the roller to intake a gamepiece.
+   *
+   * @return A command to set roller to max speed.
+   */
+  public Command roller() {
+    return setRoller(ROLLER_MAX);
```

Comment:
I doubt .4 volts will even overcome the friction to make the roller run (unless this is tested and i am trolling)